clio-server/config/site.conf

```diff
@@ -25,6 +25,7 @@ Redirect 307 / https://${SERVER_NAME}/
 ServerAlias ${SERVER_ALIAS}
 DocumentRoot /app
 
+AllowEncodedSlashes NoDecode
```

Comment:
Do you have a test that verifies that this fixes the issue? I would've thought that using `NoDecode` would require our code to explicitly decode the URL encoding.

Reply:
I've confirmed it works by deploying this branch to of Clio server to dev 
`wm28d-f87:clio yanc$ java -Dclio.server.hostname=clio.gotc-dev.broadinstitute.org -Dclio.server.port=443 -Dclio.server.use-https=true -Dclio.client.service-account-json=clio-dev-account.json -XX:GCTimeLimit=50 -XX:GCHeapFreeLimit=10 -Xmx256m -jar clio-client.jar add-gvcf --project C321 --data-type Exome --sample-alias "CEPH/UTAH PEDIGREE 1463" --location GCP --version 1 --metadata-location gvcf_metadata.json
2019-08-08T21:07:14,280Z [INFO] Successfully updated record for {"location": "GCP", "project": "C321", "data_type": "Exome", "sample_alias": "CEPH/UTAH PEDIGREE 1463", "version": 1}.
wm28d-f87:clio yanc$ 
`
whereas we would have got a 404 before deploying. But would probably still be good to see if I can add something to the integration test...

Reply:
Ah, actually the success message uses the pre-encoded values. Querying yields the encoded result... but just for the forward slash

Reply:
Ugh, but if we use `AllowEncodedSlashes On`, it's decoded before the route is parsed, so we get a different kind of 404... Will have to explicitly decode after the routing logic